react-intl/react-intl-tests.tsx

```diff
@@ -60,6 +56,17 @@ const SomeFunctionalComponentWithIntl = injectIntl<SomeComponentProps>(({
     );
 });
 
+@injectIntl
```

Comment:
Here I agree with @Kallikrein, since decorator syntax is experimental let's switch to using `injectIntl` as a function.